src/app.fetch-httpClient.config.js

```diff
@@ -70,34 +70,34 @@ export class FetchConfig {
   }
 
   /**
-   * @param {HttpClient|Rest[]} client
+   * @param {HttpClient|Rest[]} aClient
    *
    * @return {HttpClient[]}
    */
-  configure(client) {
-    if (Array.isArray(client)) {
+  configure(aClient) {
```

Comment:
client is a property already or something. i hate it if i can't search. anyways, invalid now